static/css/documentation.css

```diff
@@ -218,6 +218,12 @@ a img {
   position: relative;
 }
 
+#header-button {
+  background: #5C9A32;
```

Comment:
May want to use #7BC143

Reply:
Hm, I'm wavering back to the original color.  I think it looks better a bit darker

Reply:
I think the original color definitely makes the text easier to read.

Reply:
I've gone ahead and dropped the change to revert it back to it's previous color.

Reply:
looks good - goal was to keep it consistent with the code site, and this color matches.